lib/Quantization/Quantization.cpp

```diff
@@ -296,6 +300,30 @@ static Node *quantizeNode(Function *F, Node *node,
     quantizedNode = F->createSigmoid(SN->getName(), quantizedInputs[0]);
     break;
   }
+  case Kinded::Kind::LocalResponseNormalizationNodeKind: {
+    auto *LRN = cast<LocalResponseNormalizationNode>(node);
+    assert(quantizedInputs.size() == 1 && "Invalid number of inputs");
+    assert(qParams.size() == 1 && "Invalid number of quantized outputs");
+
+    // Note: This should either be lowered into an IntLookupTable, or
+    // implemented via a backend-specific Node/Inst.
+    quantizedNode = F->createLocalResponseNormalization(
+        LRN->getName(), quantizedInputs[0], LRN->getHalfWindowSize(),
+        LRN->getAlpha(), LRN->getBeta(), LRN->getK());
+    break;
+  }
+  case Kinded::Kind::SoftMaxNodeKind: {
+    auto *SMN = cast<SoftMaxNode>(node);
+    // SoftMax node has 2 inputs, but only one should be quantized.
+    assert(quantizedInputs.size() == 1 && "Invalid number of inputs");
+    assert(qParams.size() == 1 && "Invalid number of quantized outputs");
+
+    // Note: This should either be lowered into an IntLookupTable, or
+    // implemented via a backend-specific Node/Inst.
+    quantizedNode = F->createSoftMax(SMN->getName(), quantizedInputs[0],
```

Comment:
Do you have a plan to add softMax support to the interpreter?

Reply:
I am not sure how we can quantize SoftMax because the intermediate sum can be huge, but if some backends need it then we can go for it.

Reply:
I think @jspark1105 had some ideas about this. Where did we end up with the quantized softmax?

Reply:
We haven't quite finished our work on quantizing softmax. gemmlowp has a quantized softmax implementation: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/kernels/internal/optimized/optimized_ops.h#L3907